banyand/internal/storage/index.go

```diff
@@ -201,12 +200,11 @@ func (s *seriesIndex) Search(ctx context.Context, series []*pbv1.Series, filter
 	}
 
 	pl := seriesList.ToList()
-	if filter != nil && filter != logical.ENode {
+	if blugeQuery != nil {
 		var plFilter posting.List
 		func() {
 			if tracer != nil {
 				span, _ := tracer.StartSpan(ctx, "filter")
```

Comment:
Please add `String` to blugeQuery. The query tracing needs to debug the relevant issues.